docs/guide/mode-and-env.md

```diff
@@ -86,7 +86,7 @@ And the following `.env.staging` file:
 
 ```
 NODE_ENV=production
-VUE_APP_TITLE=My App (staging)
+VUE_APP_TITLE=My Staging App Title
```

Comment:
`My Staging App` may be better ?  Corresponds to `My App` above

Reply:
As long as the parentheses are gone.

Reply:
By the way, should I close this pr and create a new one with the change? I'm really new to the pr process.

Reply:
No, commit the change to your `patch-1` branch